src/main/java/edu/tseidler/service/Drawer.java

```diff
@@ -0,0 +1,69 @@
+package edu.tseidler.service;
+
+import edu.tseidler.model.Board;
+import edu.tseidler.model.BoardField;
+import edu.tseidler.model.Coordinates;
+import edu.tseidler.model.Fields;
+
+import java.util.Collections;
+
+public class Drawer {
+    private final int maxRow;
+    private final int widthRow;
+    private final int maxCol;
+    private final int widthCol;
+    private final StringBuilder sb = new StringBuilder();
+
+    public Drawer(Board board) {
+        maxRow = board.maxRow;
+        widthRow = Math.max(
+                Integer.toString(board.maxRow).length(),
```

Comment:
może to mierzenie długości liczby w osobną metodę wrzucić, aby było jaśniejsze co się dzieje, przy okazji osuszy się kod :)